acstools/acszpt.py

```diff
@@ -276,16 +267,17 @@ def _parse_response(self):
         # the html table. The first 6 elements in the list of all elements
         # correspond to the column names. The values after that correspond to
         # actual data for each row
-        for j in range(6):
-            for i, val in enumerate(td[j::6]):
+        BLOCK_SIZE = 6
```

Comment:
No, there is only one magic number. It corresponds to the number of columns in the `HTML` table. The outer loop shifts the starting index so that the first value in the inner loop is the column name. Then every 6th element corresponds to a member of that column.

Reply:
Something to consider for the future -- What is the maintenance cost if the backend changes on your server? Look at `astroquery`, they have rolling release now because the backends are always changing on them. I don't think `acstools` will want to go this route. If there is a way to parse this without that magic number that is resistant to backend changes, it would be better.